### ISSUE TITLE
Definition of interface SpiedMember

### ISSUE THREAD

**Issue by unlight** [henrysun918/mock-factory#4]
```
export interface SpiedMember {
    _func?: jasmine.Spy;
    _get?: jasmine.Spy;
    _set?: jasmine.Spy;
}
```
Can these properties be defined as non-nullable?

When `strictNullChecks` is set to true, following recommended usage produce error:
```
service._spy.toDataObject._func.and.returnValue({ id: null }); // Object is possible undefined
```
to avoid this error I must use following constructions, which looks dirty.
```
service._spy.toDataObject._func && service._spy.toDataObject._func.and.returnValue
```

**Comment by chuanqisun** [comment]
Thanks @unlight! #5  will address the issue. In the meanwhile, you can use a typescript [non-null assertion operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator) `!` to achieve the same result without looking too dirty:
```
service._spy.toDataObject!._func.and.returnValue
```

**Comment by chuanqisun** [comment]
`--strictNullChecks` is available in version `2.0.2`. Closing the issue.
